src/instructlab/lab.py

```diff
@@ -36,7 +35,51 @@
     import torch
 
 
-@click.group(cls=DYMGroup)
+class ExpandAliasesGroup(click.Group):
```

Comment:
It was discussed during one of the community meetings that when users input an "old" command, they should get a warning that the command will be deprecated in 2 releases (I'm making this up, will need to agree as a group). I don't see it happening here. Did I miss it?

Reply:
added